doc/README.md

```diff
@@ -189,8 +192,47 @@ paths with the highest cum weight.
 * **-svg:** Generates a report in SVG format.
 * **-web:** Generates a report in SVG format on a temp file, and starts a web
   browser to view it.
-* **-png, -jpg, -gif, -pdf:** Generates a report in these formats,
+* **-png, -jpg, -gif, -pdf:** Generates a report in these formats.
+
+### Interpreting the Callgraph
+
+*  **_cum_ value**, the value of a location plus all of its descendants, is indicated by the nodes' color.
+
+    *   Redder nodes have larger _cum_ values.
+    *   Greyer nodes have smaller _cum_ values.
+
+
+* **_flat_ value**, the value of a location itself, is indicated by the font size.
+
+    *   Nodes with larger text have larger _flat_ values.
+    *   Nodes with smaller text have smaller _flat_ values.
+
+*   **Dashed Edges**: some locations between the two connected locations were
+    removed.
+
+*   **Solid Edges**: one location directly calls the other.
+
+*   **Thicker & Redder Edges**: more resources (i.e. larger value) were used along that path.
+
+*   **Thinner & Greyer Edges**: less resources (i.e. smaller value) were used along that path.
+
+![callgraph](images/callgraph.svg)
+
+In the graph: 
```

Comment:
Word this more explicitly to refer to the example graph. I would put something like "Let's consider the following example graph" before the picture and continue with this kind of explanatory narrative in the text after the picture.

Reply:
Done.